### PR TITLE
Australian snags (Sausages sizzle)

### PR DESCRIPTION
**Pull request by sunnyjim5**
Australian classic.

### CODE REVIEW

File: src/australian-snags.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Australian snags
+
+An australian BBQ classic tridation that is simple and very easy to make. Great for parties and outdoor events
```
Comment (DGKSK8LIFE) [inline src/australian-snags.md:3]: `tridation` should be `tradition`

**Review by DGKSK8LIFE — CHANGES_REQUESTED** [review]

**Review by sunnyjim5 — COMMENTED** [review]
Spelling error fixed

**Review by DGKSK8LIFE — APPROVED** [review]

File: src/australian-snags.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+1. Chop up the onions into rings
+2. Bring the sausages to the grill, turning once every 5 mins
+3. Add oil of your choice to the hotplate then add the onions
+4. Use your tongs to toss the onions until they are golden brown
+5. When the onions are golden brown turn off the hotplate
+6. When the sausages are cooked, butter the bread then add a sausage
+7. Add tomato sauce and onions on top
+8. Enjoy and serve to everyone
+
+;tags: basic snack traditional
```
Comment (hallzy) [inline src/australian-snags.md:32]:
```suggestion
;tags: basic snack traditional australian
```

File: src/australian-snags.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+## Ingredients
+
+- Sausages (thin)
+- Onions
+- Tomato sauce
+- Mustard (optional)
+- Bread
+- Butter
+- Oil
+- Wood chips (optional, fuel for smokey flavour)
```
Comment (hallzy) [inline src/australian-snags.md:19]: How much of everything?

File: src/australian-snags.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+- ⏲️ Prep time: 2 min
+- 🍳 Cook time: 20 min
+- 🍽️ Servings: Depends
+
+## Ingredients
+
+- Sausages (thin)
+- Onions
+- Tomato sauce
+- Mustard (optional)
```
Comment (hallzy) [inline src/australian-snags.md:15]:
```suggestion
- Mustard (optional)
```

This isn't in the directions. If using, when would it be added?

File: src/australian-snags.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+An australian BBQ classic tradition that is simple and very easy to make. Great for parties and outdoor events
+especially national events. Sometimes served at hardware chains.
+
+- ⏲️ Prep time: 2 min
+- 🍳 Cook time: 20 min
+- 🍽️ Servings: Depends
+
+## Ingredients
+
+- Sausages (thin)
```
Comment (hallzy) [inline src/australian-snags.md:12]:
```suggestion
- Sausages (cut thinly)
```

**Review by hallzy — CHANGES_REQUESTED** [review]

**Pull request merged**